### PR TITLE
[OSDOCS-8885]:[OSD-GCP] Include a note on DDOS protection Service for OSD on Google Cloud

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-8885
Link to docs preview:

https://68700--docspreview.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/policy-process-security#network-security_policy-process-security
QE review:
- [x] QE has approved this change.

QE has approved. See https://issues.redhat.com/browse/OSDOCS-8885?focusedId=23595525&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23595525
Additional information:

image

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @jaybeeunix, when you get a moment could you please provide a quick review for this PR? Thanks!

File: modules/policy-security-regulation-compliance.adoc [hunk 1]
```diff
@@ -27,7 +27,7 @@ Red Hat performs periodic vulnerability scanning of {product-title} using indust
 [id="firewall_{context}"]
 === Firewall and DDoS protection
 Each {product-title} cluster is protected by a secure network configuration at the cloud infrastructure level using firewall rules (AWS Security Groups or Google Cloud Compute Engine firewall rules). {product-title} customers on AWS are also protected against DDoS attacks with link:https://docs.aws.amazon.com/waf/latest/developerguide/ddos-overview.html[AWS Shield Standard].
-
+Similarly, all GCP load balancers and public IPs used by {product-title} on GCP are protected against DDoS attacks with link:https://cloud.google.com/armor/docs/managed-protection-overview[Google Cloud Armor Standard].
```
Comment (tmalove) [inline modules/policy-security-regulation-compliance.adoc:30]:
```suggestion
Similarly, all GCP load balancers and public IP addresses used by {product-title} on GCP are protected against DDoS attacks with link:https://cloud.google.com/armor/docs/managed-protection-overview[Google Cloud Armor Standard].
```
See SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_i

**Review by tmalove — APPROVED** [review]

**Pull request merged**